kubernetes/mqtt2prometheus.yml

```diff
@@ -0,0 +1,57 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: mqtt2prometheus
+  namespace: mqtt
```

Comment:
Please create also a namespace object if you hardcode the namespace. Otherwise just remove the namespace and let the user choose the namepspace via `kubectl apply --namespace=mqtt -f kubernetes/mqtt2prometheus.yml`